routes/api/rooms.js

```diff
@@ -0,0 +1,51 @@
+var keystone = require('keystone');
+var Room = keystone.list('Room');
+var Event = keystone.list('Event');
+var _ = require('underscore');
+var moment = require('moment');
+
+exports.getAll = function (req, res) {
+	
+	//example:
+	//api/rooms?events=true&start=somedate&end=somedate
+	//Gets all rooms with the events during that timeperiod
+	
+	var rangeStart = moment(req.query.start).format();
+	var rangeEnd = moment(req.query.end).format();
+	var includeEvents = req.query.events;
```

Comment:
Do var includeEvents = !!req.query.events so this a boolean and better to check on